### PR TITLE
Add additional proximity sensors

### PR DESCRIPTION
**Pull request by mib1185**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/109630
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/proximity.markdown [hunk 1]
```diff
@@ -43,10 +42,24 @@ For each tracked device or person, a sensor is created showing the direction of
...
+</div>
+
 ### Nearest device
 
 A sensor is created showing the device or person which is nearest (_shortest distance_) to the monitored zone. If several devices or persons are at the same nearest distance, this sensor displays them all.
 
-<div class="note">
-To calculate the distance and the direction of travel for a tracked device or person, they must specify a geo-location.
-</div>
+### Nearest Distance
```
Comment (c0ffeeca7) [inline source/_integrations/proximity.markdown]:
```suggestion
### Nearest distance
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) also in headings

File: source/_integrations/proximity.markdown [hunk 2]
```diff
@@ -43,10 +42,24 @@ For each tracked device or person, a sensor is created showing the direction of
...
+
+A sensor is created showing the distance of the nearest device or person to the monitored zone in a unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection. When a tracked person or device enters the monitored zone, the distance is set to 0.
```
Comment (c0ffeeca7) [inline source/_integrations/proximity.markdown]:
```suggestion
This sensor shows the distance of the device or person nearest to the monitored zone. The unit depends on your settings for the [Home Assistant unit system](/docs/configuration/basic). When a tracked person or device enters the monitored zone, the distance is set to 0.
```

File: source/_integrations/proximity.markdown [hunk 3]
```diff
@@ -43,10 +42,24 @@ For each tracked device or person, a sensor is created showing the direction of
...
+
+### Nearest Direction of travel
+
+A sensor is created showing the direction of travel of the nearest device or person to the monitored zone. Possible states are:
```
Comment (c0ffeeca7) [inline source/_integrations/proximity.markdown]:
```suggestion
This sensor shows the direction of travel of the nearest device or person to the monitored zone. Possible states are:
```

**Review by c0ffeeca7 — COMMENTED** [review]
Some tweaks.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @mib1185 :+1:

**Comment by mib1185** [comment]
thx @c0ffeeca7 for the tweaks 👍

**Pull request merged**